_posts/2023-04-10-opensource-perf.markdown

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title:  "Collaborating to Improve Open Source Performance: A Case Study"
+authors:
+  - dblock
+  - chibenwa
+date:   2023-04-10
+categories:
+  - technical-post
+meta_keywords: OpenSearch, Apache James, Flame graph, Performance, Driver
+meta_description: Learn how key contributors of APache James found out and fixed major performance problems in OpenSearch Java driver.
```

Comment:
Apache lowercase P.
Drop "key".
Get specific on what the performance problem was (JSON parsing?)